image-filters/image-filters.markdown

```diff
@@ -3,148 +3,148 @@ author: Cate Huston
 
 _Cate Huston is a developer and entrepreneur focused on mobile. She’s lived and worked in the UK, Australia, Canada, China and the United States, as an engineer at Google, an Extreme Blue intern at IBM, and a ski instructor. Cate speaks internationally on mobile development and her writing has been published on sites as varied as Lifehacker, The Daily Beast, The Eloquent Woman and Model View Culture. She co-curates Technically Speaking, blogs at Accidentally in Code and is [\@catehstn](https://twitter.com/catehstn) on Twitter._
```

Comment:
Cate left the tech industry and spent a year finding her way back whilst building her passion project [Show & Hide](http://showandhide.com/). She is Director of Mobile Engineering at [Ride](https://ride.com/), speaks internationally on mobile development and engineering culture, co-curates [Technically Speaking](http://www.techspeak.email/) and is an advisor at [Glowforge](https://glowforge.com/). Cate doesn’t exactly live in Colombia but she spends a lot of time there, and has lived and worked in the UK, Australia, Canada, China the United States, previously as an engineer at Google, an Extreme Blue intern at IBM, and a ski instructor. Cate blogs at [Accidentally in Code](http://accidentallyincode.com/) and is [@catehstn](https://twitter.com/catehstn) on Twitter.